dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/RpcConstants.java

```diff
@@ -16,17 +16,24 @@
  */
 package org.apache.dubbo.rpc;
 
-import org.apache.dubbo.common.Constants;
+import org.apache.dubbo.common.constants.ClusterConstants;
+import org.apache.dubbo.common.constants.CommonConstants;
+import org.apache.dubbo.common.constants.ConfigConstants;
+import org.apache.dubbo.common.constants.FilterConstants;
+import org.apache.dubbo.common.constants.MonitorConstants;
+import org.apache.dubbo.common.constants.RegistryConstants;
+import org.apache.dubbo.common.constants.RemotingConstants;
 
 /**
  * RpcConstants
  *
  * @deprecated Replace to org.apache.dubbo.common.Constants
  */
 @Deprecated
-public final class RpcConstants extends Constants {
+public final class RpcConstants implements ClusterConstants, CommonConstants, ConfigConstants, FilterConstants,

```

Comment:
Why is this class not deleted, I see that it has not been used anymore.

Reply:
it's claimed to be deprecated, user may rely on it. We cannot remove it in 2.7 but we can do it in 3.0